barretenberg/cpp/src/barretenberg/vm/avm/trace/trace.cpp

```diff
@@ -1544,6 +1544,67 @@ void AvmTraceBuilder::op_calldata_copy(uint8_t indirect,
     });
 }
 
+void AvmTraceBuilder::op_returndata_size(uint8_t indirect, uint32_t dst_offset)
+{
+    auto const clk = static_cast<uint32_t>(main_trace.size()) + 1;
+    bool tag_match = true;
```

Comment:
nitpick: add comment " // This boolean will not be a trivial constant anymore once we constrain address resolution."